### DISCUSSION TITLE
Divisão de tarefas

### DISCUSSION THREAD

**Discussion by ribeiroevandro** [diego3g/rsxp-2023#23] [General]
Como será feito o controle das tarefas, visando evitar que mais de uma pessoa, trabalhe na mesma feature. Existe uma forma de "assinar" a tarefa que irei fazer?

**Comment by joaovrcosta** [comment]
Acredito que como vai ser por meio de pull requests terá como avaliar quais dessas features valem a pena serem implementadas, pq pense que alguém assine uma feature, imagina q essa pessoa demore ou essa feature seja dependência de outra, isso acarreta em problemas, então vai existir um fluxo de atividades e cada pessoa escolhe fazer conforme esse fluxo mas no final vão ser avaliadas para implementação no projeto

**Reply by diego3g** [reply]
Isso, vou criar as issues e cada um sinaliza no que está trabalhando conforme conseguir. De resto, vamos nos baseando na ordem que as PRs forem chegando :)

**Comment by diego3g** [comment]
Vou fechar essa discussão porque estamos com bastante discussões abertas com tópicos que super valem a contribuição de vocês por lá também e acho que não vale estender muito a discussão por aqui. Sintam-se livres para reabrir a discussão se fizer sentido 🤗
